.gitignore

```diff
@@ -1,2 +1,2 @@
-channels.scm
-guix-installer.iso
\ No newline at end of file
+*.iso
+./guix/channels.scm
```

Comment:
In the last force-push I manually deleted this file (i guess this is what you mean by duped?)

I also don't understand why the `.gitignore` won't work, the file still gets tracked anyway...

Reply:
Nah, I'm referring to the channel config that's written literally in `build.yaml` in the Guix installation step.

Reply:
Alright, all good then. I've answered on that specifically already, above.